RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1368,7 +1368,7 @@ boolean adjustForYellowRay(string combat_string)
 
 boolean adjustForYellowRayIfPossible(monster target)
 {
-	if(canYellowRay(target))
+	if(canYellowRay(target) && !in_pokefam()) // No YR in Pocket Familiars, so removing this from options.
```

Comment:
just make canYellowRay return false when in pokefam

Reply:
Attempted to fix this via most recent commit